run_dir/static/js/pricing_main.js

```diff
@@ -0,0 +1,994 @@
+                components[comp_id] = {
+                    'component': this.all_components[comp_id],
+                    'quantity': info['quantity']
+                }
+            }
+            return components
+        }
+    }
+}
+
+const app = Vue.createApp(vPricingMain)
+
+/* Reusable components */
+
+app.component('v-exchange-rates', {
+  /* Exchange rates used fot the cost calculator and a modal to
+   * change dates for when the rates were issued.
+   */
+  props: ['mutable', 'issued_at'],
+  data() {
+      return {
+          issued_at_form_bound: this.issued_at
+      }
+  },
+  computed: {
+      USD_in_SEK() {
+          val = this.$root.USD_in_SEK
+          if (val === null) {
+              return ""
+          } else {
+              return val.toFixed(2)
+          }
+      },
+      EUR_in_SEK() {
+          val = this.$root.EUR_in_SEK
+          if (val === null) {
+              return ""
+          } else {
+              return val.toFixed(2)
+          }
+      }
+  },
+  watch: {
+      issued_at(newVal, oldVal) {
+          this.issued_at_form_bound = newVal
+      }
+  },
+  methods: {
+    reload_exch_rates() {
+        date = this.issued_at_form_bound;
+        this.$root.fetchExchangeRates(date);
+    }
+  },
+  template:
+      /* Using the html comment tag can enable html syntax highlighting in editors */
+      /*html*/`
+      <h4>Exchange rates</h4>
+      <dl class="row">
+        <dt class="col-md-4 text-right">1 USD</dt>
+        <dd class="col-md-8"><span id='exch_rate_usd'>{{USD_in_SEK}}</span></dd>
+        <dt class="col-md-4 text-right">1 EUR</dt>
+        <dd class="col-md-8"><span id='exch_rate_eur'>{{EUR_in_SEK}}</span></dd>
+        <dt class="col-md-4 text-right">Issued at</dt>
+        <dd class="col-md-8"><span id='exch_rate_issued_at'>{{issued_at}}</span>
+          <a v-if="mutable" href="#" data-toggle="modal" data-target="#exch_rate_modal"> (Change)</a>
+        </dd>
+      </dl>
+      <template v-if="mutable">
+        <!-- Exchange Rate Modal -->
+        <div class="modal fade" id="exch_rate_modal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+          <div class="modal-dialog" role="document">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h4 class="modal-title" id="myModalLabel">Fetch historic exchange rate</h4>
+                <button type="button" class="btn-close" data-dismiss="modal" aria-label="Close"></button>
+              </div>
+              <div class="modal-body">
+                <p>The most recent rates prior to the date chosen will be used.</p>
+                <p>Latest date exchange rates: <input id="datepicker" type='date' v-model="issued_at_form_bound" data-date-format="yyyy-mm-dd"></p>
+              </div>
+              <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
```

Comment:
btn-default has no styling in bootstrap 5

Reply:
Fixed!